daemon/logger/jsonfilelog/jsonfilelog.go

```diff
@@ -301,7 +302,7 @@ func tailFile(f io.ReadSeeker, logWatcher *logger.LogWatcher, tail int, since ti
 		msg, err := decodeLogLine(dec, l)
 		if err != nil {
 			if err != io.EOF {
-				logWatcher.Err <- err
+				logWatcher.Err <- fmt.Errorf("error while tailing logs: %v", err)
```

Comment:
this will shodow the real error if anything above this function is trying to check for error types like io.EOF or os.IsNotExist